x-pack/elastic-agent/pkg/agent/application/handler_action_policy_change.go

```diff
@@ -99,6 +100,14 @@ func (h *handlerPolicyChange) handleKibanaHosts(c *config.Config) (err error) {
 			err, "fail to create API client with updated hosts",
 			errors.TypeNetwork, errors.M("hosts", h.config.Fleet.Kibana.Hosts))
 	}
+	ctx, cancel := context.WithTimeout(ctx, 15*time.Second)
```

Comment:
maybe we can use const or some config value, if you like

Reply:
added const for now